src/main/kotlin/no/nav/familie/oppdrag/repository/MellomlagringKonsistensavstemmingRepository.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.familie.oppdrag.repository
+
+import no.nav.familie.oppdrag.service.Fagsystem
+import org.springframework.data.jdbc.repository.query.Query
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+interface MellomlagringKonsistensavstemmingRepository :
+    RepositoryInterface<MellomlagringKonsistensavstemming, UUID>,
+    InsertUpdateRepository<MellomlagringKonsistensavstemming> {
+
+    fun findAllByFagsystemAndAvstemmingstidspunktAndAktiv(
+        fagsystem: Fagsystem,
+        avstemmingstidspunkt: String,
+        aktiv: Boolean
+    ): List<MellomlagringKonsistensavstemming>
+
+    @Query(
+        "SELECT COALESCE(sum(antall_oppdrag),0) from mellomlagring_konsistensavstemming " +
```

Comment:
Trenger vel ikke `COALESCE` her? 
```suggestion
        "SELECT sum(antall_oppdrag) from mellomlagring_konsistensavstemming " +
```